.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -11,7 +11,14 @@ assignees: ''
 A clear and concise description of what the bug is.
 
 **To Reproduce**
-Test case or steps to reproduce the behavior or.
+Test case or steps to reproduce the behavior or.  
+It would be appreciated if you could provide code that can be executed with as few changes as possible,
+such as including `import`.
```

Comment:
not sure what "including 'import'" here means? That code sample should include `import` statements too, not just code?

Reply:
> That code sample should include import statements too, not just code?

Yes, I intended to write that.
The reason is that the `import` target may be ambiguous and difficult to reproduce depending on the content.

Reply:
Ok, just minor addition then:

    such as include `import` statements.

Reply:
Fixed.
7e3c923